README.rst

```diff
@@ -170,7 +170,7 @@ traditionally used for convenience and simplicity.
 
 The advantages of Postgres include:
 
- * significant performance improvements due to the superior threading and
+ * significant performannce improvements due to the superior threading and
```

Comment:
if this causes pain, please suggest something better... surely postgres' multithreading should be helping us. Or are you objecting to the suggestion that performance overall has actually increased?

> On 30 Apr 2015, at 19:29, Erik Johnston <notifications@github.com> wrote:
> 
> In README.rst:
> 
> > @@ -170,7 +170,7 @@ traditionally used for convenience and simplicity.
> >  
> >  The advantages of Postgres include:
> >  
> > - * significant performance improvements due to the superior threading and
> > + * significant performannce improvements due to the superior threading and
> nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnn
> 
> —
> Reply to this email directly or view it on GitHub.
>